### PR TITLE
data sim 5/x: more test fns

### PR DESCRIPTION
**Pull request by oxade**
## Description 

Adds ability to advance checkpoint, clock, epoch

## Test Plan 

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-transactional-test-runner/src/test_adapter.rs [hunk 1]
```diff
@@ -608,6 +609,20 @@ impl<'a> MoveTestAdapter<'a> for SuiTestAdapter<'a> {
             }};
         }
         match command {
+            SuiSubcommand::AdvanceCheckpoint => {
+                self.executor.advance_epoch().await?;
```
Comment (wlmyng) [inline crates/sui-transactional-test-runner/src/test_adapter.rs:613]: should this also be advance_epoch

File: crates/sui-transactional-test-runner/src/args.rs [hunk 2]
```diff
@@ -127,6 +127,20 @@ pub struct SetAddressCommand {
     pub input: ParsedValue<SuiExtraValueArgs>,
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct AdvanceCheckpointCommand {
+    #[clap(long = "url")]
+    pub url: String,
```
Comment (wlmyng) [inline crates/sui-transactional-test-runner/src/args.rs:133]: what are url and port for

File: crates/sui-transactional-test-runner/src/test_adapter.rs (hunk 1 above)
Comment (oxade) [reply crates/sui-transactional-test-runner/src/test_adapter.rs]: indeed!

File: crates/sui-transactional-test-runner/src/args.rs (hunk 2 above)
Comment (oxade) [reply crates/sui-transactional-test-runner/src/args.rs]: copy pasta from breaking up a PR which would let us host the simulator. Fixed

**Review by wlmyng — APPROVED** [review]

**Pull request merged**